nservicebus/upgrades/5to6.md

```diff
@@ -77,3 +77,11 @@ In Version 6 we are improving the code based configuration story. For example yo
 ### They are a crutch that enabled complex configuration
 
 Configuration should be simply to apply to your code and easy to manipulate by hand. Unfortunately the configuration helpers enabled more complex xml configuration by allowing people to take the stance of "it is OK this is complex to configure since the xml can be generated". This is not a scenario we want to enable moving forward.
+
+## Sagas
+
+### Unique attribute no longer needed
+
+NServiceBus will automatically make correlated saga properties unique without the need for a explicit `[Unique]` attribute to be used. You can safely remove the attribute from your saga data types.
+
+Read more on saga and concurrency [here](/nservicebus/sagas/concurrency.md)
```

Comment:
make this a related header instead https://github.com/Particular/docs.particular.net#related